### PR TITLE
Doc/edits dpf core intro and misc

### PR DESCRIPTION
**Pull request by PipKat**
First editorial review for me in this project.

### CODE REVIEW

File: docs/source/getting_started/docker.rst [hunk 1]
```diff
@@ -4,47 +4,45 @@
 Using DPF Through Docker
 ************************
 
-You can run DPF within a container on any OS using `docker`.
+You can run DPF within a container on any OS using Docker.
```
Comment (akaszynski) [inline docs/source/getting_started/docker.rst:7]: Please provide a link to Docker here.

File: docs/source/getting_started/docker.rst [hunk 2]
```diff
@@ -54,16 +52,19 @@ directly from the command line.
...
 directories must be mapped to the ``/dpf`` directory for the server to
-see the files you wish it to read.
+see the files you want it to read.
 
 
 Using the DPF Container from Python
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Normally ``ansys.dpf.core`` attempts to start the DPF server at the first usage of a DPF class.  If you do not have ANSYS installed and simply wish to use the docker image, you can override this behavior by connecting to the DPF server on the port you mapped with:
+Normally ``ansys.dpf.core`` attempts to start the DPF server at the first 
+usage of a DPF class.  If you do not have ANSYS installed and simply want
```
Comment (akaszynski) [inline docs/source/getting_started/docker.rst:65]:
```suggestion
usage of a DPF class.  If you do not have Ansys installed and simply want 
```

File: docs/source/getting_started/index.rst [hunk 3]
```diff
@@ -26,10 +26,10 @@ Installation
    docker
    
    
-Try Installation
+Tryout Installation
```
Comment (akaszynski) [inline docs/source/getting_started/index.rst:29]: I don't really like this section.  Should just be installation.

File: docs/source/getting_started/index.rst [hunk 4]
```diff
@@ -80,30 +80,30 @@ A quick try out of the installation can be done with:
...
 
-DPF-Core dependencies are automatically checked installed when the packages is installed.
-Those dependencies are:
+DPF-Core dependencies are automatically checked when packages are 
+installed. The dependencies are:
 
-- ansys.grpc.dpf : package with grpc code generated from protobufs
-- psutil
-- progressbar2
+- `ansys.grpc.dpf` with thegRPC code generated from protobufs
```
Comment (akaszynski) [inline docs/source/getting_started/index.rst:86]: Check spaces.

Also, these packages should have links to the pypi packages.  Google psutil pypi and select the first hit.

File: docs/source/getting_started/index.rst [hunk 5]
```diff
@@ -80,30 +80,30 @@ A quick try out of the installation can be done with:
...
 
 Optional Dependencies
 ~~~~~~~~~~~~~~~~~~~~~
 
 Optional dependencies can be installed for specific usage:
 
-- matplotlib : for chart plotting
-- pyvista : for 3D plotting
-- scooby : for dependency reports
+- ``matplotlib`` for chart plotting
```
Comment (akaszynski) [inline docs/source/getting_started/index.rst:95]: Same here.

File: docs/source/user_guide/plotting.rst [hunk 6]
```diff
@@ -73,8 +74,9 @@ extract the nodal strain in the X direction.
...
 .. note::
 
-   At the moment, only fields with Elemental and Nodal locations are
-   supported.  Use the :py:meth:`to_nodal
-   <ansys.dpf.core.field.Field.to_nodal>` to convert to nodal or the
-   ``'nodal_to_elemental'`` operator to convert to elemental.
+   Only fields with elemental and dodal locations are
+   supported currently.  Use the :py:meth:`to_nodal
+   <ansys.dpf.core.field.Field.to_nodal>` operator to 
+   convert to nodal or the ``'nodal_to_elemental'`` operator
```
Comment (akaszynski) [inline docs/source/user_guide/plotting.rst:80]: Adding a link to nodal_to_elemental operator would be great.

File: docs/source/user_guide/operators.rst [hunk 7]
```diff
@@ -134,16 +144,16 @@ is used in several other examples.
...
 
     op.inputs.data_sources(data_src)
     
 Other optional inputs can be connected to the displacement operator.
-Printing the operator above showed that a mesh_scoping of type :ref:`ref_scoping`
-can be connected to work on a spatial subset. A time scoping of type list[int] 
-can also be connected to work on a temporal subset:
+Printing the operator above showed that a ``mesh_scoping`` of type :ref:`ref_scoping`
+can be connected to work on a spatial subset. A ``time_scoping`` of a list type with 
+integers can also be connected to work on a temporal subset:
```
Comment (akaszynski) [inline docs/source/user_guide/operators.rst:156]:
```suggestion
can be connected to work on a spatial subset. A ``time_scoping`` of a list of integers 
can also be connected to work on a temporal subset:
```

File: docs/source/user_guide/operators.rst [hunk 8]
```diff
@@ -155,8 +165,8 @@ can also be connected to work on a temporal subset:
     
 Evaluating Operators
 ~~~~~~~~~~~~~~~~~~~~
-With all the required inputs assigned, the fields_container can now be
-output from the operator with:
+With all the required inputs assigned, the ``fields_container`` can be
```
Comment (akaszynski) [inline docs/source/user_guide/operators.rst:168]: Please provide a link to fields_container.  @cbellot000 can help here.

File: docs/source/user_guide/operators.rst [hunk 9]
```diff
@@ -294,28 +299,33 @@ one operator to the inputs of another operator with:
...
+***************************************
+Operators for Importing or Reading Data
+***************************************
+
+These operators provide for reading data from solver files or from standard file types:
 
-- Operators importing/reading data
-- Operators transforming existing data
-- Operators exporting data
+- For MAPDL, supported solver file formats include RST, MODE, RFRQ, and RDSP.
```
Comment (akaszynski) [inline docs/source/user_guide/operators.rst:319]: This reads much better.  Thanks!

**Comment by akaszynski** [comment]
@cbellot000, I'd like you to review this when you get a chance.

File: docs/source/user_guide/plotting.rst [hunk 10]
```diff
@@ -73,8 +74,9 @@ extract the nodal strain in the X direction.
 
 .. note::
 
-   At the moment, only fields with Elemental and Nodal locations are
-   supported.  Use the :py:meth:`to_nodal
-   <ansys.dpf.core.field.Field.to_nodal>` to convert to nodal or the
-   ``'nodal_to_elemental'`` operator to convert to elemental.
+   Only fields with elemental and dodal locations are
```
Comment (PipKat) [inline docs/source/user_guide/plotting.rst:77]:
```suggestion
   Only fields with elemental and nodal locations are
```

File: docs/source/getting_started/index.rst [hunk 11]
```diff
@@ -80,30 +80,28 @@ A quick try out of the installation can be done with:
...
 Optional Dependencies
 ~~~~~~~~~~~~~~~~~~~~~
 
-Optional dependencies can be installed for specific usage:
+Optional package dependencies can be installed for specific usage:
 
-- matplotlib : for chart plotting
-- pyvista : for 3D plotting
-- scooby : for dependency reports
+- - `Mathplotlib <https://pypi.org/project/matplotlib/>`_ for chart plotting
```
Comment (cbellot000) [inline docs/source/getting_started/index.rst:95]: little typo here, Matplotlib doesn't have a "h"

File: docs/source/getting_started/install.rst [hunk 12]
```diff
@@ -3,29 +3,30 @@
 *********************
 Installation with PIP
 *********************
-Once you've installed Ansys 2021R1 or newer, you can install DPF with:
+Once Ansys 2021 R1 or later is installed, you can install the
```
Comment (cbellot000) [inline docs/source/getting_started/install.rst:6]: Could we please have a way specify here that it's from 2021 R1 to the latest version of Ansys

**Comment by cbellot000** [comment]
Thank you very much @PipKat for your documentation review, it's really helpful

File: docs/source/getting_started/install.rst (hunk 12 above)
Comment (PipKat) [reply docs/source/getting_started/install.rst:6]: @cbellot000 It already says 2021 R1 or later--that is standard Ansys phrasing....

File: docs/source/getting_started/index.rst (hunk 11 above)
Comment (PipKat) [inline docs/source/getting_started/index.rst:95]:
```suggestion
- - `Matplotlib <https://pypi.org/project/matplotlib/>`_ for chart plotting
```

Comment (PipKat) [reply docs/source/getting_started/index.rst:95]: @cbellot000 Great catch! Thanks!

File: docs/source/user_guide/operators.rst (hunk 8 above)
Comment (PipKat) [reply docs/source/user_guide/operators.rst]: @akaszynski  I think I provided a valid link.

File: docs/source/getting_started/index.rst (hunk 11 above)
Comment (PipKat) [inline docs/source/getting_started/index.rst:95]:
```suggestion
- - `Matplotlib <https://pypi.org/project/matplotlib/>`_ for chart plotting
```

File: docs/source/getting_started/install.rst (hunk 12 above)
Comment (cbellot000) [reply docs/source/getting_started/install.rst:6]: Sorry, my bad

**Review by cbellot000 — APPROVED** [review]

**Comment by akaszynski** [comment]
CI/CD failure was likely just due to the agent failing to pick up the task.  @PipKat, if you see this feel free just to rerun it to see if you can get failed runs working.  Timeouts of 60 minutes indicate that something failed to start or got locked up on Azure Devops.

**Pull request merged**